cpp/unreal_plugins/SimulationController/Source/SimulationController/UrdfRobotAgent.cpp

```diff
@@ -0,0 +1,175 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#include "SimulationController/UrdfRobotAgent.h"
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <Camera/CameraComponent.h>
+#include <Components/SceneCaptureComponent2D.h>
+#include <Engine/World.h>
+#include <Math/Rotator.h>
+#include <Math/Vector.h>
+
+#include "CoreUtils/ArrayDesc.h"
+#include "CoreUtils/Assert.h"
+#include "CoreUtils/Config.h"
+#include "CoreUtils/Log.h"
+#include "CoreUtils/Std.h"
+#include "CoreUtils/Unreal.h"
+#include "SimulationController/CameraSensor.h"
+#include "UrdfRobot/UrdfLinkComponent.h"
+#include "UrdfRobot/UrdfRobotComponent.h"
+#include "UrdfRobot/UrdfRobotPawn.h"
+
+UrdfRobotAgent::UrdfRobotAgent(UWorld* world)
+{
+    SP_LOG_CURRENT_FUNCTION();
```

Comment:
Our convention is to only call `SP_LOG_CURRENT_FUNCTION` in entry points called by Unreal.

Reply:
Updated the code.